### DISCUSSION TITLE
Add application_name to PostgreSQL connection

### DISCUSSION THREAD

**Discussion by munnik** [grafana/grafana#67616] [Data Sources]
It would be great if Grafana can add a (configurable) application_name to the PostgreSQL [connection string](https://www.postgresql.org/docs/15/libpq-connect.html). This is especially helpful when one wants to analyze database perfomance with tools like [pgBadger](https://github.com/darold/pgbadger).

**Comment by munnik** [comment]
If wrote the changes for the .go files in https://github.com/fuelessence/grafana/commit/bb76808ae65f1613c22a12f2738d4cf27a89413c but I'm not able to do the changes in the Javascript/Angular code.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82750. Feel free to continue the discussion around this there. Thank you!
